jenkins-tools/fetch-jobs.rb

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ruby
+#
+# This script get jobs configurations. It relies on jenkins_api_client gem
+# instead of Jenkins Java CLI, so it's quiet fast.
+#
+#   ruby fetch-jobs.rb --config ci-jenkins.yml --prefix yast- --output configs
+#
+# You can get more information using the --help option:
+#
+#   ruby fetch-jobs.rb --help
+begin
+  require "jenkins_api_client"
+rescue LoadError
+  STDERR.puts "You need to install rubygem-jenkins_api_client package"
```

Comment:
NP: You should prefer `$stderr` instead, `STDERR` is a constant and cannot be redefined while `$stderr` is a global variable which can be modified at runtime (you can easily redirect the output or you can capture the output and verify it in tests...).